config/sync/views.view.right_sidebar_latest_revision.yml

```diff
@@ -393,6 +434,250 @@ display:
           entity_field: nid
           plugin_id: node_nid
       display_extenders: {  }
+      css_class: translate-button
+    cache_metadata:
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url
+        - 'user.node_grants:view'
+      tags: {  }
+  attachment:
+    display_plugin: attachment
+    id: attachment
+    display_title: 'Translations button'
+    position: 6
+    display_options:
+      display_extenders: {  }
+      title: ''
+      defaults:
+        title: false
+        fields: false
+        filters: false
+        filter_groups: false
+        relationships: false
+      fields:
+        nid:
+          id: nid
+          table: node_field_revision
+          field: nid
+          relationship: none
+          group_type: group
+          admin_label: ''
+          label: ''
+          exclude: false
+          alter:
+            alter_text: true
+            text: '<a href="/node/{{ nid__value }}/translations" class="button">Manage translation</a>'
```

Comment:
This introduces inconsistency between "Translate" action on dashboard and "Manage translation" on right sidebar of node view.